### PR TITLE
fix model loss

### PR DESCRIPTION
**Pull request by TCChenlong**
修复loss相关的内容

### CODE REVIEW

File: paddle2.0_docs/image_classification/mnist_lenet_classification.ipynb [hunk 1]
```diff
@@ -209,11 +199,9 @@
     "            loss = paddle.nn.functional.cross_entropy(predicts, y_data)\n",
     "            # 计算损失\n",
     "            acc = paddle.metric.accuracy(predicts, y_data, k=2)\n",
```
Comment (jzhang533) [inline paddle2.0_docs/image_classification/mnist_lenet_classification.ipynb:201]: 这里可以改成k=1
k=2的话很快就100%了。
